__device-tests__/helpers/utils.js

```diff
@@ -223,7 +223,15 @@ const toggleHtmlMode = async ( driver: wd.PromiseChainWebdriver ) => {
 		// hit Enter
 		await driver.pressKeycode( 66 );
 	} else {
-		// TODO: toggle html mode in iOS
+		const menuButton = await driver.elementByAccessibilityId( '...' );
+		await menuButton.click();
+		let toggleHtmlButton;
+		try {
+			toggleHtmlButton = await driver.elementByAccessibilityId( 'Switch to HTML' );
+		} catch ( err ) {
+			toggleHtmlButton = await driver.elementByAccessibilityId( 'Switch To Visual' );
```

Comment:
Trying to understand the logic, is the assumption here that `'Switch to HTML'` tap will fail only if we are already in html mode? If so, let's add a comment in the code about it. Reason is that since we'll be expecting the `try` to fail in half the cases, this is not an error case after all but rather, the way we detect which menu option is currently available. WDYT?

Reply:
I went ahead and wrote the extra code to get this more descriptive without the comments. Initially, it seemed the simpler way was just to go with the `try catch` but it's not that much additional effort to make this work without the assumption  so I did that